types/src/traits/node_implementation.rs

```diff
@@ -301,7 +303,32 @@ pub type CommitteeMembership<TYPES, I> =
         <I as NodeImplementation<TYPES>>::Leaf,
         Message<TYPES, I>,
     >>::Membership;
-
+// TODO remove this really ugly type once we have `ConsensusExchanges` impl and can use that to do all generation
+/// Type for the underlying `ConnectedNetwork` that will be shared (for now) b/t Communication Channels
```

Comment:
With the `ConsensusExcahnges`, I added types that are similarly ugly, just shorter than this. 😅